presto-main/src/main/java/com/facebook/presto/operator/scalar/MathFunctions.java

```diff
@@ -107,6 +161,44 @@ public static double ceiling(@SqlType(StandardTypes.DOUBLE) double num)
         return Math.ceil(num);
     }
 
+    private static SqlScalarFunction decimalCeilingFunction(String name)
+    {
+        Signature signature = Signature.builder()
+                .kind(SCALAR)
+                .name(name)
+                .literalParameters("num_precision", "num_scale")
+                .argumentTypes("decimal(num_precision, num_scale)")
+                .returnType("decimal(num_precision, num_scale)")
```

Comment:
based on mysql behaviour the return type should be `decimal(num_precision-num_scale+1, 0)`